apps/src/aichat/views/ChatWorkspace.tsx

```diff
@@ -65,10 +68,24 @@ const ChatWorkspace: React.FunctionComponent<ChatWorkspaceProps> = ({
 
   useEffect(() => {
     if (selectedStudent) {
-      dispatch(fetchStudentChatHistory(selectedStudent.id));
+      dispatch(
+        fetchUserChatHistory({userId: selectedStudent.id, isOwnHistory: false})
+      );
     }
   }, [selectedStudent, currentLevelId, dispatch]);
 
+  useEffect(() => {
```

Comment:
Should these be one `useEffect`? Basically describing the sequence of events that needs to happen when either the level, selected student, or current user changes? (clear messages, fetch the appropriate chat history if necessary)

Reply:
Good call -- combined them!